Source/JavaScriptCore/parser/Parser.cpp

```diff
@@ -2080,7 +2057,7 @@ template <class TreeBuilder> TreeStatement Parser<LexerType>::parseStatement(Tre
         // These tokens imply the end of a set of source elements
         return 0;
     case ESCAPED_KEYWORD:
-        if (!matchAllowedEscapedContextualKeyword()) {
+        if (UNLIKELY(!matchAllowedEscapedContextualKeyword())) {
```

Comment:
I suppose we don't even need the `break;` inside this block, since it's an unconditional `fail`.
(Seems like there are probably a handful of cases like this.)

Reply:
Nice, done!